package.json

```
@@ -14,10 +14,10 @@
   },
   "author": "SilverStripe Ltd",
   "engines": {
-    "node": "^10.x"
+    "node": "^18.x"
   },
   "scripts": {
-    "build": "yarn && yarn lint && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && yarn lint && yarn test && NODE_ENV=production webpack --mode production --bail --progress",
```

```suggestion
    "build": "yarn && yarn lint && yarn test && rm -rf client/dist/* && NODE_ENV=production webpack --mode production --bail --progress",
```
Remove everything in dist before we start, so we know we don't have any extra garbage.